src/ontology/mondo-ingest.Makefile

```diff
@@ -764,6 +765,16 @@ $(EXTERNAL_CONTENT_DIR)/mondo-otar-subset.robot.tsv:
 $(EXTERNAL_CONTENT_DIR)/mondo-medgen.robot.tsv:
 	wget "https://github.com/monarch-initiative/medgen/releases/latest/download/medgen-xrefs.robot.template.tsv" -O $@
 
+###### MalaCards #########
+
+# Managed by support@genecards.org
+
+$(TMPDIR)/malacards.tsv:
+	wget "https://genecardscustomers.blob.core.windows.net/mondo/Mondo_MalaCards_Current.tsv" -O $@
+
+$(EXTERNAL_CONTENT_DIR)/mondo-malacards.robot.tsv: $(TMPDIR)/malacards.tsv
+	awk 'NR==1 { print; print "ID	AI obo:mondo#curated_content_resource	>A oboInOwl:source"; next } { print }' $< > $@
+
```

Comment:
did you want to set the output file as `.PRECIOUS` similar to the others?

Reply:
Yes, thanks, fixed!